### ISSUE TITLE
builds(deps): add vcsim to rpm

### ISSUE THREAD

**Issue by smitterl** [vmware/govmomi#3413]
**Is your feature request related to a problem? Please describe.**
It completes the accepted change https://github.com/vmware/govmomi/pull/3403

**Describe the solution you'd like**
The govmomi package should include the vcsim binary, too.

**Describe alternatives you've considered**
None

**Comment by smitterl** [comment]
@dougm Thanks for accepting my change. I just noticed that vcsim is currently not part of the package. Slippage on my side because I don't use it. However, the package name 'govmomi' implies reasonably that all binaries should be included. So, I'll fix this.
If you think it makes sense to also build 'deb' packages for Debian derived distros, please let me know. It's easily added.

**Comment by dougm** [comment]
That's cool, but let's wait on deb packages and we can add if requested.
